tests/integration/gov/genesis_test.go

```diff
@@ -204,23 +204,24 @@ func TestImportExportQueues_ErrorUnconsistentState(t *testing.T) {
 	suite := createTestSuite(t)
 	app := suite.app
 	ctx := app.BaseApp.NewContext(false)
-	require.Panics(t, func() {
```

Comment:
if we are expecting a panic here then why do we change it to require no err. This cause the test to fail